### ISSUE TITLE
[bug][android][newarch] Exception in HostFunction: Loss of precision during arithmetic conversion: (long)

### ISSUE THREAD

**Issue by lovegaoshi** [Sharcoux/slider#102]
**Describe the bug**
`accessibilityValue` in the `Thumb` component breaks in RN 0.76.1 +  newarch + android, with `Exception in HostFunction: Loss of precision during arithmetic conversion: (long)`
I looked into the accessibilityValue type and it states min, max and now values should be integers. but slider will pass them as double instead, causing this exception.

**To Reproduce**
this is built with RN's reproducer template. Couldnt figure out how to use newarch with snack, sorry:S
https://github.com/lovegaoshi/slider-bug

**Expected behavior**
probably wrap them with Math.round will do the trick

**Screenshots**
N/A

**Please complete the following information:**
RN 0.76.1 +  newarch + android

**Additional context**
N/A

**Comment by Sharcoux** [comment]
Ok, but what should we do for floating values? Like, a slider with a step of 0.5 for instance...

**Comment by lovegaoshi** [comment]
i saw the snippet at react-native-paper and they just round up to a percentage value
```

    <View
      onLayout={onLayout}
      {...rest}
      accessible
      accessibilityRole="progressbar"
      accessibilityState={{ busy: visible }}
      accessibilityValue={
        indeterminate ? {} : { min: 0, max: 100, now: progress * 100 }
      }
```
though i just got a crash from converting 14.00000000000002. LOL

**Comment by Sharcoux** [comment]
Lol
Ok, so what should I do? ^^

**Comment by lovegaoshi** [comment]
how does
accessibilityValue={
        { min: 0, max: 100, now: math.round(progress * 100) }
      }
sound ?

**Comment by Sharcoux** [comment]
Ok, I'll just do this for decimal sliders in order to maintain expected behavior for other use cases.

**Comment by Sharcoux** [comment]
Ok, released in version 8.0.5
